test/distributed/fsdp/test_checkpoint_wrapper.py

```diff
@@ -40,6 +48,61 @@ def test_load_activation_checkpointed_module(self):
         for p1, p2 in zip(lin.parameters(), lin_new.parameters()):
             self.assertEqual(p1, p2)
 
+    @skip_if_lt_x_gpu(1)
+    def test_checkpoint_wrapper_parity(self):
+        """
+        Tests that using checkpoint_wrapper or the functional
+        torch.utils.checkpoint (with the same reentrant config)
+        results in the same maximum memory usage, i.e. they are
+        equivalent memory usage wise.
+        """
+        class Model(nn.Module):
+            def __init__(self, n: int, use_cp: bool, use_wrapper: bool = False, use_reentrant=True):
```

Comment:
nit: Can we clean this up to be consistent?
For example:
```suggestion
            def __init__(self, n: int, use_cp: bool, use_wrapper: bool = False, use_reentrant: bool = True):
```